.vuepress/public/postman/meilisearch-collection.json

```diff
@@ -986,6 +991,70 @@
 								}
 							},
 							"response": []
+						},
+						{
+							"name": "Get sortable attributes",
+							"request": {
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{url}}/indexes/{{indexUID}}/settings/sortable-attributes",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"indexes",
+										"{{indexUID}}",
+										"settings",
+										"sortable-attributes"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Update sortable attributes",
+							"request": {
+								"method": "POST",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "[\"price\"]"
+								},
+								"url": {
+									"raw": "{{url}}/indexes/{{indexUID}}/settings/sortable-attributes",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"indexes",
+										"{{indexUID}}",
+										"settings",
+										"sortable-attributes"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Reset sortable attributes Copy",
```

Comment:
is the `Copy` at the end of the `name` field supposed to be there?

Reply:
Oh! Well spotted